webapp/components/youtube_video.jsx

```diff
@@ -79,12 +79,10 @@ export default class YoutubeVideo extends React.Component {
 
     componentDidMount() {
         if (global.window.mm_config.GoogleDeveloperKey) {
-            $.ajax({
-                async: true,
-                url: 'https://www.googleapis.com/youtube/v3/videos',
-                type: 'GET',
-                data: {part: 'snippet', id: this.state.videoId, key: global.window.mm_config.GoogleDeveloperKey},
-                success: this.handleReceivedMetadata
+            request.get('https://www.googleapis.com/youtube/v3/videos').
```

Comment:
I think this should be it's own function. Maybe in web_client so we don't have superagent spread across the code.